src/vendor/webauthn.js

```diff
@@ -0,0 +1,185 @@
+"use strict";
```

Comment:
Where is this file coming from? Did we write it, or is it a thirdparty file?

If we wrote it, what's it doing in vendor?

If we didn't write it:
- Where is it coming from? Can we get it from NPM so we don't need to check it into source?
- What is the license? Are we modifying it (and if we are, where)?